### PR TITLE
added cmd/armadactl/README.md

### LINKED ISSUE #2258: Add readme for each armanda component in the cmd folder
Trying to understand the codebase as a newbie poses a bit of difficulty especially as some components do not have any documentation in the docs folder. It would really be helpful if there was a readme for each armada component in the cmd directory.

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
What type of PR is this?
Documentation

What this PR does / why we need it:
added cmd/armadactl/README.md

Which issue(s) this PR fixes:
Fixes #2258

Special note for the reviewer: Thanks a lot for spending your precious time :)

### CODE REVIEW

**Comment by akash-kumar-saw** [comment]
Hiii @kannon92,

Can you please review this PR and can approve it !!!

Thanks in Advance : )

File: cmd/armadactl/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Armadactl
+
+Armadactl is a command line tool for managing and deploying Armada charts. It allows users to install, upgrade, and delete charts, as well as view their status and history.
```
Comment (kannon92) [inline cmd/armadactl/README.md:3]: Not sure what you mean here.  Armadactl is for interacting with armada objects via the cli ie creating queues, submitting jobs, watching events.  NOt sure what you mean by charts or status or history.

**Comment by akash-kumar-saw** [comment]
Hi @kannon92,

I have changed the first paragraph as per your suggestions ...
Now you can review it and can approve it : )

Thanks

File: cmd/armadactl/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# Armadactl
+
+Armadactl is a command-line tool used for managing jobs in the Armada workload orchestration system. It provides functionality for creating, updating, and deleting jobs, as well as monitoring job status and resource usage.
+
+## Installation
+To install Armada, follow these steps:
+
+- Install Armada CLI:
+```bash
+curl -sL https://github.com/armadaplatform/armada-cli/releases/latest/download/armada-linux-amd64 -o armada
```
Comment (kannon92) [inline cmd/armadactl/README.md:10]: I don't think this link is correct.

File: cmd/armadactl/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    namespace: armada
+    tiller_namespace: kube-system
+    tiller_host: localhost:44134
+    helm:
+        timeout: 300
+    manifests:
+        path: /etc/armada
+```
+
+## Contributing
```
Comment (kannon92) [inline cmd/armadactl/README.md:101]: I don't think we need these files.  They are already present in the README.

Comment (akash-kumar-saw) [reply cmd/armadactl/README.md:101]: You are taking about which file??

Comment (akash-kumar-saw) [reply cmd/armadactl/README.md:101]: the whole cmd/armadactl/README.md!!!

Comment (kannon92) [reply cmd/armadactl/README.md:101]: These sections I mean.  From this line and one down I think we can just remove.

Comment (kannon92) [reply cmd/armadactl/README.md:101]: #Contributing and #License are not necessary for this readme.

Comment (akash-kumar-saw) [reply cmd/armadactl/README.md:101]: ohhh...
ookkk... for sure...

File: cmd/armadactl/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Armadactl is a command-line tool used for managing jobs in the Armada workload orchestration system. It provides functionality for creating, updating, and deleting jobs, as well as monitoring job status and resource usage.
+
+## Pre-requisites
+- Git
+- Docker
+- Helm v3.5+
+- Kind v0.11.1+
+- Kubectl
+
+## Installation
```
Comment (kannon92) [inline cmd/armadactl/README.md:12]: Can you remove this installation section?

If you want to add a README.md about `armadactl` and this cmd I think that is fine.  

But I would ask that we don't replicate text that is somewhere else in the repo.  It is hard to keep docs up to date.

Comment (akash-kumar-saw) [reply cmd/armadactl/README.md:12]: I guess you are right... I am removing the whole installation section...

**Comment by akash-kumar-saw** [comment]
Hi @kannon92,

I have revised the README file according to your suggestions...
You can review it now and can approve it : )

Thanks !!!

File: cmd/armadactl/README.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+armadactl getQueueSchedulingReport
+```
+- getJobSchedulingReport : This subcommand retrieves a report of the current scheduling status of all jobs in the Armada cluster.
+```bash
+armadactl getJobSchedulingReport
+```
+
+For a full list of subcommands and options, you can run **armadactl --help**.
+
+## Configuration
```
[the remark this answers is not in the document]
Comment (kannon92) [reply cmd/armadactl/README.md:71]: Yea I think removing is a good idea.

**Comment by akash-kumar-saw** [comment]
Hi @kannon92,

Now I guess it is perfect to be merged...
You can review it now and can approve it : )

Thanks !!!

**Review by kannon92 — APPROVED** [review]

**Comment by akash-kumar-saw** [comment]
Hii @kannon92 ,

I guess I have committed all changes in master branch... Will that cause any problems... Should I delete this PR and should create new one with new branch??? or should I leave it as it is???

**Comment by kannon92** [comment]
> Hii @kannon92 ,
> 
> I guess I have committed all changes in master branch... Will that cause any problems... Should I delete this PR and should create new one with new branch??? or should I leave it as it is???

Nope. I would recommend a branch next time but it’s fine. Just merged.

**Comment by akash-kumar-saw** [comment]
Hiii @kannon92,

Thanks for spending your precious time in reviewing this PR : )

**Pull request merged**